spec/views/confirmations/show.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "confirmations/show.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
We usually don't add specs for views. You can delete this file

Reply:
view will be removed